src/Telemetry/Opt_In/Status.php

```diff
@@ -67,44 +64,38 @@ public function get_option() {
 	 *
 	 * @since 1.0.0
 	 * @since 2.0.1 Correct logic so it is not subject to the order of the plugins.
+	 * @since TBD   Update to remove unnecessary "mixed" status.
 	 *
-	 * @return integer The status value.
+	 * @return bool The site's current opt in status.
```

Comment:
I think we need to keep this an integer in order to keep the same method signature and not cause a breaking change.